Tiltfile

```diff
@@ -541,7 +541,7 @@ def deploy_clusterclass(clusterclass_name, label, filename, substitutions):
     delete_clusterclass_cmd = kubectl_cmd + " --namespace=$NAMESPACE delete clusterclass " + clusterclass_name + ' --ignore-not-found=true; echo "\n"'
 
     local_resource(
-        name = clusterclass_name,
+        name = clusterclass_name + ".clusterclasses",
```

Comment:
Should this be singular?

Reply:
took the wording from the label below, but agree.